app/components/works/work_type_modal_component.html.erb

```diff
@@ -11,7 +11,7 @@
   <button type="button" class="btn-close" data-dismiss="modal" aria-label="Close"></button>
 </div>
 <div class="modal-body">
-  <form method="get" action="/collections/1/work/new" data-target="work-type.form">
+  <form method="get" action="<%= collection_path %>" data-target="work-type.form">
```

Comment:
I'm not sure I understand why this needed to change.

Reply:
Without changing this here, I was unable to get correct action is not set when reusing this model for the collection show. So the button always goes to `/collections/1/work/new` instead of the actual path for a new work in a collection. During pairing with @mjgiarlo this was our first pass at a possible solution.

Any idea why it wouldn't work as is in this context?

Reply:
This is also the source of the type error. So perhaps there's a better way to do this.

Reply:
I just think this is unnecessary, as https://github.com/sul-dlss/happy-heron/blob/main/app/javascript/controllers/work_type_controller.js#L9 takes care of overwriting this. I think this change is a no-op, but it adds extra code which we don't need to support, so I think it should be reverted.